beacon-chain/state/state-native/spec_parameters.go

```diff
@@ -0,0 +1,32 @@
+package state_native
+
+import (
+	"fmt"
+
+	"github.com/prysmaticlabs/prysm/config/params"
+	"github.com/prysmaticlabs/prysm/runtime/version"
+)
+
+func (b *BeaconState) ProportionalSlashingMultiplier() (uint64, error) {
+	switch b.version {
+	case version.Bellatrix:
+		return params.BeaconConfig().ProportionalSlashingMultiplierBellatrix, nil
+	case version.Altair:
+		return params.BeaconConfig().ProportionalSlashingMultiplierAltair, nil
+	case version.Phase0:
+		return params.BeaconConfig().ProportionalSlashingMultiplier, nil
+	}
+	return 0, fmt.Errorf("ProportionalSlashingMultiplier() not supported for state version %d", b.version)
```

Comment:
You can use https://github.com/prysmaticlabs/prysm/blob/0d2696ed4ec33dc362bc68e68137251aa5273a79/beacon-chain/state/state-native/types.go#L34-L36